tools/dendrite_openapi_version

```diff
@@ -1,2 +1,2 @@
-COMMIT="f3810e7bc1f0d746b5e95b3aaff32e52b02dfdfa"
-SHA2="3a54305ab4b1270c9a5fb0603f481fce199f3767c174a03559ff642f7f44687e"
+COMMIT="b425a6c783b3081613ffa00407f271cae06568c4"
```

Comment:
This commit is on a branch, right? Is that OK? For example, what happens when we squash and merge the PR in Dendrite? I was under the impression that commit no longer exists at all, but I could be wrong about that.

I sort of expected this PR to require no changes to Dendrite, but I'm not sure if that's possible. For example, one could do this:

- Add the SMF properties in a Dendrite PR, in which they're unused. Merge that PR to main.
- Pull that commit on Dendrite's `main` in here, and then set the properties. They're still unused.
- Then update Dendrite to now use those SMF properties.

I guess it depends on how much we want to guarantee that Omicron `main` always works against Dendrite `main`. @Nieuwejaar may have a suggestion on that front. I think it's not crucial that we can always run `main` from both, but I do think we need to not break developer workflows if we're pointing to a non-existent commit. Could we double-check that?

Reply:
So, from what I gather once the images are built, the commits last for some time. Of course, the goal of this would be to merge this, then dendrite, and then have a separate PR here to align it with the updated commit.